app/src/main/java/net/gsantner/markor/activity/DocumentShareIntoFragment.java

```diff
@@ -99,17 +103,28 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         } else {
             _shareIntoImportOptionsFragment = (ShareIntoImportOptionsFragment) getChildFragmentManager().findFragmentByTag(ShareIntoImportOptionsFragment.TAG);
         }
+
         _hlEditor.setText(sharedText);
-        _hlEditor.setBackgroundColor(ContextCompat.getColor(view.getContext(), as.isDarkThemeEnabled() ? R.color.dark__background_2 : R.color.light__background_2));
-        _hlEditor.setTextColor(ContextCompat.getColor(view.getContext(), as.isDarkThemeEnabled() ? R.color.white : R.color.dark_grey));
+        _hlEditor.setBackgroundColor(ContextCompat.getColor(context, as.isDarkThemeEnabled() ? R.color.dark__background_2 : R.color.light__background_2));
+        _hlEditor.setTextColor(ContextCompat.getColor(context, as.isDarkThemeEnabled() ? R.color.white : R.color.dark_grey));
         _hlEditor.setTextSize(TypedValue.COMPLEX_UNIT_SP, as.getFontSize());
         _hlEditor.setTypeface(Typeface.create(as.getFontFamily(), Typeface.NORMAL));
 
-        if (sharedText.trim().isEmpty()) {
+        if (sharedText.isEmpty() || sharedText.equals(prefix)) {
             _hlEditor.requestFocus();
         }
     }
 
+    private static String formatOrPrefix(final String format, final String value) {
+        try {
+            final String result  = String.format(format, value);
+            if (!result.equals(format) && !TextUtils.isEmpty(value)) {
+                return result;
+            }
+        } catch (IllegalFormatException ignored) { }
+        return format + " " + value;
+    }
```

Comment:
```suggestion
    private static String formatOrPrefix(String format, final String value) {
        format = format.contains("%s") ? format : (format + "%s");
        return format.replace("%s", TextUtils.isEmpty(value) ? "" : value);
    }
```

Reply:
I thought of this, but felt that there were edge cases with escaped '\%s' etc. In the end It's only a couple lines extra